index.html

```diff
@@ -1,346 +1,463 @@
-						>
-					</div>
-					<div class="category-about_page">
-						<header
-							class="about_page__header flex-row justify-between align-center"
-						>
-							<h3 class="about_page__subtitle">Languages</h3>
-							<img src="img/about_page/arr_d.svg" alt="Arrow down icon" />
-						</header>
-						<ul class="about_page__list grid">
-							<li class="about_page__item">
-								<img
-									src="img/about_page/javascript.png"
-									alt="javascript png image"
-								/>
-								JavaScript
-							</li>
-							<li class="about_page__item">
-								<img
-									src="img/about_page/html.png"
-									alt="javascript png image"
-								/>
-								HTML
-							</li>
-							<li class="about_page__item">
-								<img
-									src="img/about_page/css.png"
-									alt="javascript png image"
-								/>
-								CSS
-							</li>
-						</ul>
-					</div>
-					<div class="category-about_page">
-						<header
-							class="about_page__header flex-row justify-between align-center"
-						>
-							<h3 class="about_page__subtitle">Frameworks</h3>
-							<img
-								src="img/about_page/arr_r.svg"
-								alt="Arrow right icon"
-							/>
-						</header>
-					</div>
-					<hr />
-					<div class="category-about_page">
-						<header
-							class="about_page__header flex-row justify-between align-center"
-						>
-							<h3 class="about_page__subtitle">Skills</h3>
-							<img
-								src="img/about_page/arr_r.svg"
-								alt="Arrow right icon"
-							/>
-						</header>
-					</div>
-					<hr />
-				</div>
-			</section>
-			
-   <section class="contact-me">
-    
-    <form>
-      <div class="background">
-      <h2>contact-me</h2>
-      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
-        I’d love to help with it!</p>
-      <label for="">
-        <input type="text" id="message" placeholder="Lorem Ipsum"/>
-      </label>
-      <label for="">
-        <input type="text" id="message" placeholder="loremipsum@gmail.com"/>
-      </label>
-      <textarea id="message-box" placeholder = "">
-        
-      </textarea>
-      <button class="el-botones">
-        <p>Get in touch</p>
-      </button>
-      </div>
-    </form>
-  </section>
-  </div>
-
-
-		</main>
-	</body>
-</html>
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap");
+
+:root {
+  /* Primary Colors */
+  --CL-primary-blue: #6070ff;
+  --CL-primary-blue-250: #7f8cff;
+  --CL-primary-blue-500: #396df2;
+  --CL-primary-blue-600: #2230d2;
+  --CL-primary-gray-300: #7a869a;
+  --CL-primary-gray-650: #344563;
+  --CL-primary-light: #ebebff;
+  --CL-primary-dark: #172b4d;
+
+  /* Neutral Colors */
+  --CL-neutral-white: #fff;
+  --CL-neutral-white-off: #dfe1e6;
+  --CL-neutral-white-off-200: #f7f7f9;
+
+  /* Font Families */
+  --FF-primary-poppins: "Poppins", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
```

Comment:
- [ ] You have a `CSS` codebase inside the `index.html` file which is meant to render you milestone contact form as an additional feature to your previous milestones.
- [ ] Kindly add the appropriate `HTML` code while meeting this milestone's [Project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md)

## Requirements:
![image](https://github.com/AsciencioAlex/Personal-Portfolio/assets/50990925/da0d6ef5-99c4-4c28-8ebf-6f3d370a9009)